### ISSUE TITLE
Support `null`

### ISSUE THREAD

**Issue by spilist** [jackfranklin/test-data-bot#39]
It doesn't seem to be possible to set a certain field as `null`. 

A sample code:
```
import { build, fake, bool } from 'test-data-bot';

const Account = build('Account').fields({
  trialStartedAt: null,
});

export default Account;
```

It causes 
```
    TypeError: Cannot read property 'name' of null
    at Generator.fullyExpandReturn (node_modules/test-data-bot/src/generator.js:14:22)
```
, due to the following line. `value` doesn't have a name.
```
// generator.js
    } else if (value.name === 'builderFuncToReturn') {
      return value()
```

**Comment by jackfranklin** [comment]
Ah that's definitely something we should fix!

In the mean time, I think this would work:

```
trialStartedAt: perBuild(() => null)
```

But we should support directly setting null. PR welcome if you fancy having a go :)

**Comment by jackfranklin** [comment]
Fixed in 0.7.1

**Comment by spilist** [comment]
Oh, I was trying to make a PR, but it's very quickly fixed :) Thanks!

**Comment by jackfranklin** [comment]
No worries! I had a few minutes spare this morning to fix :) #5 is a good one if you do fancy doing a PR - but no probs if not.
